portfolio/src/main/webapp/index.html

```diff
@@ -15,28 +16,13 @@ <h2>Click here for a random fact about me:</h2>
             <button onclick="addRandomGreeting()">Random Fact</button>
             <div id="greeting-container"></div>
       </div>
-      <div id=" pictures">
-          <!--photo one -->
-            <img width="300px"  height="300px" src="/images/imageOne.jpg"/>
-
-         <!--photo two -->
-            <img width="300px"  height="300px" src="/images/imageTwo.jpg"/>
-
-         <!--photo three -->
-            <img width="300px"  height="300px" src="/images/imageThree.jpg"/>
-      </div>
-
-      <div id="pictures_row_2">
-
-         <!--photo Four -->
-            <img width="300px"  height="300px" src="/images/imageFour.jpg"/>
-
-         <!--photo Five -->
-            <img width="300px"  height="300px" src="/images/imageFive.jpg"/>
-
-         <!--photo six -->
-            <img width="300px"  height="300px" src="/images/imageSix.jpg"/>
-      </div>
+        <!--row of photos -->
+            <img class="pictures" width="300px"  height="300px" alt="image one: sunset" src="/images/imageOne.jpg"/>
```

Comment:
One cool thing you can do instead of adding the "pictures" class to every image element is to add it to the single parent element instead. Then in the css, you can use the `.pictures img` selector.